browser/net/brave_network_delegate_base.cc

```diff
@@ -93,20 +115,80 @@ int BraveNetworkDelegateBase::OnHeadersReceived(net::URLRequest* request,
   // URLRequestHttpJob::awaiting_callback_ will be set to true after we
   // return net::ERR_IO_PENDING here, callbacks need to be run later than this
   // to set awaiting_callback_ back to false.
-  BrowserThread::PostTask(BrowserThread::IO, FROM_HERE,
+  base::PostTaskWithTraits(
+      FROM_HERE, {BrowserThread::IO},
       base::Bind(&BraveNetworkDelegateBase::RunNextCallback,
         base::Unretained(this), request, ctx));
   return net::ERR_IO_PENDING;
 }
 
+bool BraveNetworkDelegateBase::OnCanGetCookies(const URLRequest& request,
```

Comment:
As it turns out we weren't blocking all 3p cookies from the network service.  This fixes it.